crates/rolldown/src/stages/link_stage/bind_imports_and_exports.rs

```diff
@@ -424,7 +443,7 @@ impl BindImportsAndExportsContext<'_> {
       let is_external = matches!(self.index_modules[rec.resolved_module], Module::External(_));
       if is_esm && is_external {
         if let Specifier::Literal(ref name) = named_import.imported {
-          if name.as_str() != "default" && is_validate_identifier_name(name) {
+          if is_validate_identifier_name(name) {
```

Comment:
Are we going to to support non valid identifier in the future?

Reply:
in theory yes.